wca-guidelines.md

```diff
@@ -225,6 +225,7 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-H><multiple-blindfolded><multipleblindfoldedsolving> Article H: Multiple Blindfolded Solving
 
+- H1+) [ADDITION] If a duplicate cube is found and during an attempt, the cube may be re-scrambled using an unused scramble, at the discretion of the WCA Delegate.
```

Comment:
Multi-Blind only exists for 3x3x3 right now, but we usually use "puzzle" when possible.
```suggestion
- H1+) [ADDITION] If a duplicate puzzle is found during an attempt, the cube puzzle be re-scrambled using an unused scramble, at the discretion of the WCA Delegate.
```